_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -947,12 +1005,13 @@ Now that we've learned how impactful effective data visualization can be, and ho
 
 Pair up with another learner and discuss your data and what type of exploratory data analysis you would like to perform.
 
-Questions to conisder:
+Questions to answer:
 
-1. What information do you have in your data set?
-1. Are there specific questions are you interested in investigating with the data you have?
-1. What visuals can you create to explore your data? Draw some out on paper! What will these plots show you?
-1. Is your data in a format that can easily be read into R? How can you improve the formatting of the contents of the file? 
+1. In 1-2 sentences, describe the information covered in your dataset.
+2. How large is your dataset? How many rows? How many columns?
+3. Write down 3 questions you could answer with your data set.
+4. How can you use visuals to explore those questions? Draw 3 plots out on paper. What will these plots show you?
+5. Is your data in a format that can easily be read into R? How can you improve the formatting of the contents of the file? 
```

Comment:
I think I included something like this originally, but I'm a bit worried that they'll go modifying the excel file, which we don't want. So maybe we should remove this one? 

We should also be clear in the worksheet that they shouldn't modify the excel file